src/main/kotlin/org/rust/lang/core/psi/RsCodeFragment.kt

```diff
@@ -26,20 +26,23 @@ abstract class RsCodeFragment(
     fileViewProvider: FileViewProvider,
     contentElementType: IElementType,
     open val context: RsElement,
-    forceCachedPsi: Boolean = true
-) : RsFileBase(fileViewProvider), PsiCodeFragment {
+    forceCachedPsi: Boolean = true,
+    val importTarget: RsElement? = null
```

Comment:
```suggestion
    val importTarget: RsElement? = null,
```

Reply:
I haven't seen this style applied elsewhere (e.g. `RsPathCodeFragment` in the same file), is trailing comma in multiline parameters the correct style?

Reply:
Kotlin compiler started to support trailing comma only few months ago (somewhere in September), so it technically can't be used in `RsPathCodeFragment`. Also I am not aware of any consensus regarding the style, but it seems reasonable to use it in new code for better `git blame` (e.g. I [use](https://github.com/intellij-rust/intellij-rust/blob/44d8830e1c9f396137120a908d69bcb2387946b2/src/main/kotlin/org/rust/lang/core/resolve2/CrateDefMap.kt#L334-L337) it)

Reply:
Ah, now I remember, one of the most awaited features of Kotlin 1.4 :D Ok, I'll try to remember to use it. The default formatter doesn't apply it, which is a shame though, can we change it as you have recently changed the formatting of multiline parameters?.

Reply:
We discussed and decided that it is not that important to think about it